MyLibs/fishAnalysis.h

```diff
@@ -151,6 +151,14 @@ void rot90CW(cv::Mat src, cv::Mat dst);
 double getPt2LineDistance(cv::Point2f P, cv::Point2f A, cv::Point2f B);
 /* Find 2 intersection points of a line (AB) and contour */
 std::vector<int> findPtsLineIntersectContour(std::vector<cv::Point>& contour, cv::Point2f A, cv::Point2f B);
+/* Find center of a set of points */
+cv::Point findCenter(std::vector<cv::Point>& contour);
+/* This function return two midpoint of a minBoundingBox */
+bool findMidpoint(std::vector<cv::Point>& contour, std::vector<cv::Point>& midPoint, bool isLong);
```

Comment:
Is it necessary to have this function? if you just want two midpoints of a min-box, you can simply get them by adding them together and dividing by 2.

Reply:
This function can return two midpoints from longer or shorter sides by change `bool isLong`